src/task3.java

```diff
@@ -0,0 +1,18 @@
+public class task3 {
+
+    class Person {
```

Comment:
Person class exposes its fields (name and age) directly
it's a good practice in java to encapsulate these fields with private access and provide public **getter** and **setter** methods for accessing and modifying them